netcontrold/tests/lib/test_util.py

```diff
@@ -18,71 +18,170 @@
 
 from netcontrold.lib import util
 
-_BASIC_CPU_INFO_0 = """
-core id 		: 0
-processor		: 5
-processor		: 0
-core id 		: 0
-processor		: 1
-physical id 	:
-core id 		: xyz
-processor		: 2
+# Create dumy file to fool function
+
+_BASIC_CPU_INFO_Negative = """
+core id         : 0
+processor       : 0
+physical id     : 0
+core id         : 1
+processor       : 1
+physical id     : 0
+"""
+
+_BASIC_CPU_INFO_Positive = """
+processor       : 0
+core id         : 0
+physical id     : 0
+
+processor       : 1
+core id         : 0
+physical id     : 0
+
+processor       : 2
+core id         : 1
+physical id     : 0
+
+processor       : 3
+core id         : 1
+physical id     : 0
+"""
+
+_BASIC_CPU_INFO_Processorid_NULL = """
+processor       : 
+physical id     : 0
+core id         : 1
+processor       : 
+physical id     : 0
 """
 
-_BASIC_CPU_INFO_1 = """
-processor		: 0
-core id 		: 0
-physical id 	: 0
 
-processor		: 1
-core id 		: 1
-physical id 	: 0
+_BASIC_CPU_INFO_coreid_NULL = """
```

Comment:
same as _BASIC_CPU_INFO_0_coreid ? The reason I quote the other one is that, we check for missing value there as well. So, one missing value is enough to raise the exception.